org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/panel/InfoGeneralPanel.java

```diff
@@ -490,29 +497,49 @@ private boolean initInfoTable ()
 		}
 		ArrayList<ColumnInfo> list = new ArrayList<ColumnInfo>();
 		StringBuilder sqlc = new StringBuilder().append(
-			"SELECT c.ColumnName, c.AD_Reference_ID, c.IsKey, f.IsDisplayed, c.AD_Reference_Value_ID, c.ColumnSql, c.AD_Column_ID "
+			"SELECT c.ColumnName, c.AD_Reference_ID, c.IsKey, ");	// 1-3
+		if(hasWindowAndTab) {	// 4
+			sqlc.append(" f.IsDisplayed, ");
+		} else {
+			sqlc.append(" 'Y', ");
+		}
+		sqlc.append(" c.AD_Reference_Value_ID, c.ColumnSql, c.AD_Column_ID "	// 5-7
 			+ "FROM AD_Column c"
-			+ " INNER JOIN AD_Table t ON (c.AD_Table_ID=t.AD_Table_ID)"
-			+ " INNER JOIN AD_Tab tab ON (t.AD_Table_ID=tab.AD_Table_ID)"
-			+ " INNER JOIN AD_Field f ON (tab.AD_Tab_ID=f.AD_Tab_ID AND f.AD_Column_ID=c.AD_Column_ID) "
-			+ "WHERE t.AD_Table_ID=? "
-			+ " AND tab.IsSortTab='N'"
-			+ " AND tab.Ad_Tab_ID=(SELECT MIN(mt.AD_Tab_ID) FROM AD_tab mt WHERE mt.AD_Window_ID=? AND mt.AD_Table_ID=t.AD_Table_ID AND mt.IsActive='Y')"
-			+ " AND (c.IsKey='Y' OR "
-				+ " (f.IsEncrypted='N' AND f.ObscureType IS NULL)) "
-			+ " AND c.IsActive = 'Y' "
+			+ " INNER JOIN AD_Table t ON (c.AD_Table_ID=t.AD_Table_ID)");
+		if(hasWindowAndTab) {
+			sqlc.append(" INNER JOIN AD_Tab tab ON (t.AD_Table_ID=tab.AD_Table_ID)"
+					+ " INNER JOIN AD_Field f ON (tab.AD_Tab_ID=f.AD_Tab_ID AND f.AD_Column_ID=c.AD_Column_ID) ");
+		}	
+		sqlc.append( "WHERE t.AD_Table_ID=? ");
+		if(hasWindowAndTab) {
+			sqlc.append(" AND tab.IsSortTab='N' "
+					+ " AND tab.Ad_Tab_ID=(SELECT MIN(mt.AD_Tab_ID) FROM AD_tab mt WHERE mt.AD_Window_ID=? AND mt.AD_Table_ID=t.AD_Table_ID AND mt.IsActive='Y')"
+					+ " AND (c.IsKey='Y' OR "
+					+ "		(f.IsEncrypted='N' AND f.ObscureType IS NULL))");
+		} else {
+			sqlc.append(" AND (c.IsKey='Y' "
+					+ "			OR IsIdentifier='Y' "
+					+ "			OR IsParent='Y' "
+					+ "			OR IsSelectionColumn='Y' "
+					+ "			OR Upper(ColumnName) IN ('NAME','VALUE','DESCRIPTION','DOCUMENTNO') "
+					+ "			OR Upper(ColumnName) Like '%_NAME' "
+					+ "			OR Upper(ColumnName) Like '%_Value') ");
+		}
```

Comment:
hi @PeterTakacs300 , please use c.<ColumnName> instead of just <ColumnName> (c.IsIdentifier, c.IsParent, etc)

Reply:
change committed